src/credentials/DeviceAttestationVerifier.h

```diff
@@ -213,7 +213,8 @@ class DeviceAttestationVerifier
      */
     virtual void VerifyAttestationInformation(const ByteSpan & attestationInfoBuffer, const ByteSpan & attestationChallengeBuffer,
                                               const ByteSpan & attestationSignatureBuffer, const ByteSpan & paiDerBuffer,
-                                              const ByteSpan & dacDerBuffer, const ByteSpan & attestationNonce,
+                                              const ByteSpan & dacDerBuffer, const ByteSpan & attestationNonce, VendorId vendorId,
+                                              uint16_t productId,
                                               Callback::Callback<OnAttestationInformationVerification> * onCompletion) = 0;
```

Comment:
Let's create an issue to to refactor this argument list into a single POD structure as a follow-up since this is becoming much too large of an argument list

Reply:
https://github.com/project-chip/connectedhomeip/issues/14758